libs/crypto/keys_manager.hpp

```diff
@@ -34,31 +42,31 @@ namespace iroha {
      */
     virtual bool createKeys() = 0;
 
-    /**
-     * Load plain-text keys associated with the manager, then validate loaded
-     * keypair by signing and verifying signature of test message
-     * @return nullopt if no keypair found locally, or verification failure;
-     *         related keypair otherwise
-     */
-    virtual boost::optional<iroha::keypair_t> loadKeys() = 0;
-
     /**
      * Create keys a new keypair and store it encrypted on disk
      * @param pass_phrase is a password for the keys
      * @return false if create account failed
      */
     virtual bool createKeys(const std::string &pass_phrase) = 0;
 
+    /**
```

Comment:
Why did you change the order of functions?

Reply:
It seems more logical to have overloaded function in one place.